docs/create-your-own-dapp.md

```diff
@@ -1,5 +1,21 @@
 # Creating Your First nOS dApp
 
+### Pre-requisites:
+
+First of all you'll need to build nOS browser locally, follow these instructions here:
```

Comment:
Good additions would be to **stress** that you need the nOS client to open your dApp and where you can get it.